### PR TITLE
Non-EVM chains operator-ui support

### PR DESCRIPTION
**Pull request by george-dorin**

### CODE REVIEW

File: core/web/resolver/resolver_test.go [hunk 1]
```diff
@@ -173,7 +173,7 @@ func RunGQLTests(t *testing.T, testCases []GQLTestCase) {
 	for _, tc := range testCases {
 		tc := tc
 		t.Run(tc.name, func(t *testing.T) {
-			t.Parallel()
+			//t.Parallel()
```
Comment (jmank88) [inline core/web/resolver/resolver_test.go:176]: Restore?

Comment (george-dorin) [reply core/web/resolver/resolver_test.go]: 🤦  Restored in https://github.com/smartcontractkit/chainlink/pull/11386/commits/3ee7f3042da13414a491a88f3bb4e6083379bfd2

File: core/web/resolver/query.go [hunk 2]
```diff
@@ -238,24 +244,25 @@ func (r *Resolver) Node(ctx context.Context, args struct{ ID graphql.ID }) (*Nod
...
 		}
-		return nil, err
 	}
 
-	npr, warn := NewNodePayloadResolver(&node, nil)
-	if warn != nil {
-		r.App.GetLogger().Warnw("Error creating NodePayloadResolver", "name", name, "err", warn)
-	}
+	r.App.GetLogger().Errorw("resolver getting node status", "err", chains.ErrNotFound)
+	npr, _ := NewNodePayloadResolver(nil, chains.ErrNotFound)
```
Comment (jmank88) [inline core/web/resolver/query.go:265]: OK to ignore? If so, maybe leave a comment why.

Comment (george-dorin) [reply core/web/resolver/query.go]: Fixed in https://github.com/smartcontractkit/chainlink/pull/11386/commits/9c0a2fe39e1447a40b9cc7bfd7489db2f396458f

**Review by chudilka1 — DISMISSED** [review]

**Review by jmank88 — APPROVED** [review]

**Pull request merged**